timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -627,9 +656,11 @@ export default {
       leftDrawer: true,
       expandedRows: [],
       timeFilterMenu: false,
+      addManualEvent: false,
       saveSearchMenu: false,
       saveSearchFormName: '',
       fromSavedSearch: false,
+      datetime: '',
```

Comment:
Could we be a bit more specific in the naming of this variable? i.e. someone stumbling upon this component would have a hard time figuring out that this is to be used exclusively for adding new events.